resources/js/src/app/helper/StickyElement.js

```diff
@@ -112,6 +112,10 @@ export class StickyElement
 
     checkElement(skipOffsetCalculation)
     {
+        if (isNullOrUndefined(this.el) || isNullOrUndefined(this.placeholder))
+        {
+            return;
```

Comment:
die Funktion this.updateStyles() (Z. 107) wird trotzdem ausgeführt und führt zu JavaScript Fehlern.
Man könnte hier false zurückgeben und am Ende der Funktion true.
In Zeile 106 dann eine if-Abfrage um die Funktion checkElement